.eslintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "brightspace/node-config"
+    "extends": "brightspace/node-config",
+    "env": {
+        "mocha":true
```

Comment:
This should be added in an eslintrc in your test folder

e.g. https://github.com/Brightspace/node-auth/blob/master/packages/node_modules/brightspace-auth-keys-redis-store/test/.eslintrc.yml